cylc/uiserver/scripts/gui.py

```diff
@@ -26,10 +26,11 @@
 from contextlib import suppress
 from glob import glob
 import os
-from pathlib import Path
-import random
 import re
+from requests.exceptions import RequestException
+import requests
```

Comment:
`requests` is currently coming in to cylc environment as it is required by jupyterhub, Sphinx and metomi-rose.
Is this sufficient or should I be adding it as a dependency of cylc-uiserver for completeness? Not sure if this could cause conflicts with pinning for the other packages so I haven't added it for now. Reviewer thoughts?

Reply:
As suggested at team meeting I have added this as a loose dependency with no pinning to ensure no conflict with other package requirements.